spec/spec_test.go

```diff
@@ -207,10 +207,16 @@ var _ = Describe("Interal Kit", func() {
 		RuntimeConfig: "dns",
 		CPI:           "aws", //"gcp",
 	})
-	// Test(Environment{
-	// 	Focus:       true,
-	// 	Name:        "nfs-volume-services",
-	// 	CloudConfig: "aws",
-	// 	CPI:         "aws",
-	// })
+	Test(Environment{
+		Name:          "isolation-segments",
+		CloudConfig:   "aws",
+		RuntimeConfig: "dns",
+		CPI:           "aws",
+	})
+	Test(Environment{
+		Focus:       true,
```

Comment:
This was commented out, and it no longer is.  If you're uncommenting it, you need to remove the Focus: true line, or it will be the only test run.

Reply:
Ah my bad, I will comment it out as it was.